### PR TITLE
Added DocVue Software

### PR DESCRIPTION
**Pull request by Securenetology**
Added DocVue Software. Fix for CVE-2021-4104

Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [x] Status: please select a value from the status table at the top
  - [x] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [x] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -1083,6 +1083,7 @@ The `Version` relates to the `Status` column. If `Status` field is set to 'Vulne
 | Devolutions | All products | | Not vuln | Not vuln | Not vuln | Not vuln | | [source](https://blog.devolutions.net/2021/12/critical-vulnerability-in-log4j/) |
 | DirectAdmin | All products | | Not vuln | Not vuln | Not vuln | Not vuln | Invidivual plugins not developed as part of DirectAdmin core may be vulnerable. | [source](https://forum.directadmin.com/threads/new-zero-day-exploit-for-log4j-java-library-is-an-enterprise-nightmare.65173/post-339723) |
 | Docker | Docker infrastructure | Unknown | Not vuln | Not vuln | Not vuln | Not vuln | Docker infrastructure not vulnerable, Docker images could be vulnerable. For more info see source. |[source](https://www.docker.com/blog/apache-log4j-2-cve-2021-44228/) |
+| Docvue | Docvue Client Agent | fix | Unknown | Unknown | Unknown | Unknown | New client package can be retrieved from support. | |
```
Comment (maertsen) [inline software/README.md:1086]: - Can you provide a software version in the `version` column?
  - Can you put _Fix_ in the columns that match the respective CVE?
  - Can you provide an url to a vendor statement?

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @Securenetology, thanks for contributing to the software list. Can you have a look at my comments? Some changes are necessary before we can merge.

File: software/README.md (hunk 1 above)
Comment (Securenetology) [reply software/README.md:1086]: Like this? Sorry I'm new at this.

| Docvue | Docvue Client Agent | <11.1.6.10447 | fix | Unknown | Unknown | Unknown | Unknown | New client package can be retrieved from support. | [source] https://user-images.githubusercontent.com/89155495/146959966-a3d39148-0b76-44a4-8418-5e91fa40ca2a.png |

Comment (tschmidtb51) [reply software/README.md:1086]: No worries. We are here to help you: Let's do that step by step:
1. Is `11.1.6.10447` the first fixed version? If so, just remove `<` upfront.
2. Which one of the 4 vulnerability is addressed by that patch?
  
    |Status CVE-2021-4104 | Status CVE-2021-44228 | Status CVE-2021-45046 | Status CVE-2021-45105|
    |-- | -- | -- | --|
  
    Put into the corresponding column a _Fix_, leave other empty. (I couldn't find out from the statement you linked, but maybe you have more information.)

3. Is the statement you linked publicly available / from the website or was it in an email? In the first case, please add the link to the official announcement (not only the picture; as the statement might change). In the latter one, please add a picture (which does not contain PII) to the PR in the folder vendor-statements. See [2Brightsparks](https://github.com/NCSC-NL/log4shell/blob/main/software/vendor-statements/2Brightsparks.png) as example.

If you have any question, just ask them :-)

Comment (Securenetology) [reply software/README.md]: Thanks! I think I got it! Unfortunately, all they gave me was an email. I redacted some company information.

Comment (tschmidtb51) [reply software/README.md]: I see two problems here:
1. Unfortunately, your email does not mention which of the vulnerabilities was fixed nor the exact version.
2. It is currently unclear which one of the Docvue companies you meant. The best way to clarify that is to include the email address (at least the domain name) into the picture. See [2Brightsparks](https://github.com/NCSC-NL/log4shell/blob/main/software/vendor-statements/2Brightsparks.png) how it should look like.
3. (But that is not a problem as you can upload the picture together with your PR.) The picture is not located inside the PR.

**Comment by Securenetology** [comment]
Vendor statement

DV

File: software/README.md (hunk 1 above)
Comment (Securenetology) [reply software/README.md]: Updated with vendor statement and added correct vendor.

Comment (Securenetology) [reply software/README.md]: All fixed!

File: software/README.md [hunk 2]
```diff
@@ -3452,6 +3452,7 @@ The `Version` relates to the `Status` column. If `Status` field is set to 'Vulne
 | Wind River | Wind River Linux | > 8 | Not vuln | Not vuln | Not vuln | Not vuln | no support for log4j | [source](https://support2.windriver.com/index.php?page=security-notices&on=view&id=7191)|
 | Wireshark | Wireshark | | Not vuln | Not vuln | Not vuln | Not vuln | | [source](https://www.wireshark.org/news/20211215.html)|
 | WitFoo | WitFoo Precinct | 6.x | Not vuln | Fix |  | | WitFoo Streamer & Apache Kafka Docker containers are/were vulnerable | [source](https://www.witfoo.com/blog/emergency-update-for-cve-2021-44228-log4j/)|
+| WolfPack Softwae | Docvue Client Agent and Server Agent | 11.1.6.10447 | Fix | Investigation | Investigation | Investigation | New client package can be retrieved from support. | [source](vendor-statements/WolfPack-Docvue.png)  |
```
Comment (tschmidtb51) [inline software/README.md:3455]:
```suggestion
| WolfePak Software | Docvue Client Agent and Server Agent | 11.1.6.10447 | Fix | Investigation | Investigation | Investigation | New client package can be retrieved from support. | [source](vendor-statements/WolfPack-Docvue.png) |

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
@Securenetology Thank you for your contribution.

I have a few remarks:
1. The name of the company seems to be `WolfePak`. Please adopt the name of the picture.
2. Please redact the names from the email addresses. (redacting anything before the @wolfepak.com)
3. Please redact the name from the starting "Copied on this email".
4. It is still unclear to me whether the fix is regarding the usage of Log4j 1.x or 2.x. Is there anyway you can clarify that?

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
@Securenetology  Thank you for your contribution.

Due to Github limits, we needed to reorganize the repo. Please close this PR and re-submit your changes.

The comments from this PR are still valid. Please address them in your new PR.

**Comment by dev-NCSC-NL** [comment]
Last contact was 24 day's ago. I'm closing this one.

**Pull request closed without merging**